model/ProctorioApiService.php

```diff
@@ -53,6 +53,7 @@ class ProctorioApiService extends ConfigurableService
 
     //Prefix
     public const PREFIX_KEY_VALUE = 'proctorio::';
+    public const PREFIX_DELIVERY_KEY_VALUE = self::PREFIX_KEY_VALUE . 'deliveryId::';
```

Comment:
```suggestion
    private const PREFIX_DELIVERY_KEY_VALUE = self::PREFIX_KEY_VALUE . 'deliveryId::';
```
Constant only used here.

Reply:
done